lib/src/IndexingPolicy.cpp

```diff
@@ -37,15 +37,17 @@ using namespace web::json;
 IndexingPolicy::IndexingPolicy()
 	: automatic_(true)
 	, indexing_mode_(IndexingMode::CONSISTENT)
-	, excluded_paths_(0)
+	, excluded_paths_()

```

Comment:
why remove 0 here?

Reply:
When I saw the old source, I mistook it for a mistake. The old sources are:

IndexingPolicy :: IndexingPolicy ()
: automatic_ (true)
, indexing_mode_ (IndexingMode :: CONSISTENT)
, excluded_paths_ ()
, included_paths_ (0)